### ISSUE TITLE
Countdown on website not accounting for timezone

### ISSUE THREAD

**Issue by dbolton** [nextcloud/nextcloud.com#1222]
The countdown at https://nextcloud.com/countdown/ shows over2 hours till the big reveal, yet the blog post seems to indicate that the keynote already happened: https://nextcloud.com/blog/the-new-standard-in-on-premises-team-collaboration-nextcloud-hub/

It looks like the website countdown uses local time instead of the time zone for Berlin.

**Comment by jospoortvliet** [comment]
yes, it does, and I sadly dunno how to fix that - I'm not very technical :D

will try better next time.
